.github/workflows/housekeeping.yml

```diff
@@ -0,0 +1,52 @@
+name: Housekeeping
+
+on:
+  # triage new issues
+  issues:
+    types: [opened]
+  pull_request:
+
+env:
+  PROJECT_URL: "https://github.com/orgs/celestiaorg/projects/10"
```

Comment:
Question: Will that project board become the default for epics or will we still just collect all issues here?

Reply:
this was just for the example, each repo can define their own `housekeeping.yml` flow that determines their own project boards they want issues and/or PRs assigned to.

so there is no impact to other repos by defining this here. 

this workflow is showing how to add PRs with the `project` label to project number 10. 

You can even define multiple projects with a matrix to assign issues to multiple projects if a team has a repo specific project board and then a team level project board.